TOOLS/gen-interface-changes.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+# Generate a new interface-changes.rst based on the entries in
+# the interface-changes directory.
+
+#
+# This file is part of mpv.
+#
+# mpv is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# mpv is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with mpv.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import pathlib
+import sys
+
+def add_new_entries(docs_dir, out):
+    changes_dir = pathlib.Path(docs_dir) / "interface-changes"
+    files = [f for f in pathlib.Path(changes_dir).glob("*.txt") if f.is_file()]
+    files.sort()
+    for file in files:
+        if file.name == "example.txt":
+            continue
+        with open(file.resolve(), "r") as f:
+            for line in f:
+               line =  "    - " + line.rstrip()
+               out.write(line + "\n")
+
+if __name__ == "__main__":
+    if len(sys.argv) < 2:
+        print("mpv version number argument missing")
```

Comment:
```suggestion
        print("mpv version number argument missing")
        exit(1)
```

Reply:
also maybe print general usage print(f'Usage: {sys.argv[0]} <version>)` or something

Reply:
That's nicer. Done.